src/app/tests/suites/certification/Test_TC_ACE_1_5.yaml

```diff
@@ -0,0 +1,183 @@
+      command: "readAttribute"
+      attribute: "CurrentFabricIndex"
+      response:
+          saveAs: th2FabricIndex
+
+    - label: "TH1 writes ACL giving view privilge for descriptor cluster"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value:
+                [
+                  {
+                      FabricIndex: th1FabricIndex,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [ 112233 ],
+                      Targets:
+                          [
+                              { Cluster: 0x001f, Endpoint: 0, DeviceType: null }
+                          ]
+                  },
+                  {
+                      FabricIndex: th1FabricIndex,
+                      Privilege: 1, # view
+                      AuthMode: 2, # case
+                      Subjects: null,
+                      Targets:
+                          [
+                              { Cluster: 0x001d, Endpoint: 0, DeviceType: null }
+                          ]
+                  },
+                ]
+
+    - label: "TH2 writes ACL giving view privilge for basic cluster"
+      identity: beta
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value:
+                [
+                  {
+                      FabricIndex: th2FabricIndex,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [ 112233 ],
+                      Targets:
+                          [
+                              { Cluster: 0x001f, Endpoint: 0, DeviceType: null }
+                          ]
+                  },
+                  {
+                      FabricIndex: th2FabricIndex,
+                      Privilege: 1, # view
+                      AuthMode: 2, # case
+                      Subjects: null,
+                      Targets:
+                          [
+                              { Cluster: 0x0028, Endpoint: 0, DeviceType: null }
+                          ]
+                  },
+                ]
+
+    - label: "TH1 reads descriptor cluster - expect SUCCESS"
+      command: "readAttribute"
+      cluster: "Descriptor"
+      attribute: "DeviceTypeList"
+
+    - label: "TH1 reads basic cluster - expect UNSUPPORTED_ACCESS"
+      command: "readAttribute"
+      cluster: "Basic Information"
+      attribute: "VendorID"
+      response:
+        error: UNSUPPORTED_ACCESS
+
+    - label: "TH2 reads descriptor cluster - expect UNSUPPORTED_ACCESS"
+      identity: "beta"
+      command: "readAttribute"
+      cluster: "Descriptor"
+      attribute: "DeviceTypeList"
+      response:
+        error: UNSUPPORTED_ACCESS
+
+    - label: "TH2 reads basic cluster - expect SUCCESS"
+      identity: "beta"
+      command: "readAttribute"
+      cluster: "Basic Information"
+      attribute: "VendorID"
+
+    - label: "TH1 resets ACL to default"
```

Comment:
In the test plan this comes after the RemoveFabric.  Is there a reason they are reversed here?

Reply:
yeah, it's wrong in the test plan. Please see https://github.com/CHIP-Specifications/chip-test-plans/pull/2388. the controller needs to give itself permission to do the RemoveFabric before doing it.